### PR TITLE
Responsive design

### PR DESCRIPTION
**Pull request by dipbazz**
**This project is about building a home page about the next web.**

The features of this project are listed below:
1. The website is completely responsive with three different breakpoints as:
    - Below 768px for mobile screen
    - In between 768px and 1024px for medium screen.
    - Above 1024px for large screen.

2. Website has a header:
    - With the next web logo.
    - Top header with useful links
    - Bottom header with website menu links, with a search icon.

3. Contains the main section with:
    - Banner news article
    - Latest news
    - Funding Source
    - Other news suggestions
    - Latest deals

4. Has a footer with
    - Social share buttons, Other useful links
    - Bottom footer with copyright info.

### CODE REVIEW

File: assets/css/style768.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  grid-gap: 36px 18px;
+}
+
+.news-article {
+  flex-direction: column;
+}
+
+.news-article img {
+  width: 100%;
+  height: unset !important;
```
Comment (umairarshadbutt) [inline assets/css/style768.css]: remove `!important`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,795 @@
...
+  <link rel="stylesheet" href="assets/css/style768.css" media="screen and (min-width: 768px)">
+  <link rel="stylesheet" href="assets/css/style1024.css" media="screen and (min-width: 1024px)">
+
+  <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+
+  <title>Responsive design clone of the next web homepage.</title>
+</head>
+<body>
+  <header>
+    <figure class="logo"><img src="assets/img/tnw.svg" alt=""></figure>
```
Comment (umairarshadbutt) [inline index.html]: This is just an example but there are other images that are missing a description on the alternative text attribute.
Actual
`<figure class="logo"></figure>`
** Improved** +1 :
`<figure class="logo">TNW Logo</figure>`
Having an alternate text help SEO and Screen-readers

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @dipbazz and smcommits
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] Write a descriptive alternative text on `alt=""`, don't leave it blank. See inline comment.
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by arslanbisharat — APPROVED** [review]
## Congratulations Approved 🎉
--------------------------------------------------------------------
:clap:
Hello Developer 👋
You are on 🔥
Thank you so much for making those changes and I am very happy to approve the project. You have done a great job. 👏
Please keep working hard and move forward with your curriculum.
Looking forward to seeing your next project. 😄
---------------------------------------------------------------------------------------------------------------
**To highlight**
- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**